x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/MlDailyMaintenanceService.java

```diff
@@ -88,11 +102,7 @@ private static TimeValue delayToNextTime(ClusterName clusterName) {
         int minutesOffset = random.ints(0, MAX_TIME_OFFSET_MINUTES).findFirst().getAsInt();
 
         ZonedDateTime now = ZonedDateTime.now(Clock.systemDefaultZone());
-        ZonedDateTime next = now.plusDays(1)
-            .toLocalDate()
-            .atStartOfDay(now.getZone())
-            .plusMinutes(30)
-            .plusMinutes(minutesOffset);
+        ZonedDateTime next = now.plusSeconds(10);
```

Comment:
why was this changed to just 10 seconds into the future?

Reply:
Ooops, for debugging of course.
Fixed.